zcash_primitives/CHANGELOG.md

```diff
@@ -55,6 +55,8 @@ and this library adheres to Rust's notion of
   - `builder::Builder` now has a generic parameter for the type of progress
     notifier, which needs to implement `sapling::builder::ProverProgress` in
     order to build transactions.
+  - `builder::Builder::{new, new_with_rng}` now take an additional `PaddingRules`
+    argument.
```

Comment:
`Builder::new_with_rng` has been removed.
```suggestion
  - `builder::Builder::new` now take a `BuildConfig` argument instead of an
    optional Orchard anchor.
```